DataFormats/Detectors/ITSMFT/MFT/src/TrackParamMFT.cxx

```diff
@@ -0,0 +1,370 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file TrackParamMFT.cxx
+/// \brief Implementation of the MFT track parameters for internal use
+///
+/// \author Philippe Pillot, Subatech; adapted by Rafael Pezzi, UFRGS
+
+#include "DataFormatsMFT/TrackParamMFT.h"
+
+#include <iomanip>
+#include <iostream>
+
+#include <TMath.h>
+
+#include <FairMQLogger.h>
+
+//#include "MFTTracking/Cluster.h"
+
+namespace o2
+{
+namespace mft
+{
+
+using namespace std;
+
+//_________________________________________________________________________
```

Comment:
As far as I understand, the TrackParamMFT is supposed to be used only for internal, while the TrackMFT will get its own data members after the validation of new track model, right? 
Then, I think the TrackParamMFT should not be in the DataFormats directory since there we collect classes which are supposed to be messaged over the DPL. I.e. the TrackMFT should stay in the DataFormats and become asap the meseagable (POD) class, while the TrackParamMFT it is better to move to the MFT algorithms directory. Also, since at the moment it is a base class of the TrackMFT which must be persistent, I think you should add ClassDefNV to it.

Reply:
Yes, Ruben, you are right. The inheritance from TrackParamMFT was a dirty solution to get the track at the dataformat directory. I will follow your suggestion.

Reply:
If due to the dependencies you need to keep the TrackParamMFT in the DataFormats, we can keep it at the moment there until you detach them. Anyway, if you keep the inheritance, the TrackMFT will not be messageable.

Reply:
Hi @shahor02 , TrackMFT has been cleaned-up and TrackParamMFT is back to where it belongs on the MFT Tracking directory.